src/store/getters.ts

```diff
@@ -265,14 +331,26 @@ export default {
     const { state } = rootGetterContext(context);
     const { fiatRates } = state;
     return (asset: AssetType, balance: BigNumber): BigNumber | null => {
-      if (fiatRates && fiatRates[asset] && balance) {
+      if (fiatRates && fiatRates[asset] && balance.gt(0)) {
```

Comment:
shall we check if balance exists? i.e. `balance?.gt(0)`

Reply:
good catch, awesome, thank you! I saw somewhere it return '0' if there is no balance but extra checks doesn't hurt :D